pkg/router/template/template_helper.go

```diff
@@ -335,6 +338,7 @@ func clipHAProxyTimeoutValue(val string) string {
 	if err != nil {
 		return val
 	}
```

Comment:
BTW, it seems silly to reparse `haproxyMaxTimeout` every time `clipHAProxyTimeoutValue` is called.  Why not declare it as a package variable?
```go
// haproxyMaxTimeoutDuration is haproxyDefaultTimeout as a time.Duration value.
var haproxyMaxTimeoutDuration = func() time.Duration {
	d, err := time.ParseDuration(haproxyMaxTimeout)
	if err != nil {
		panic(err)
	}
	return d
}()
```